examples/bert/README.md

```diff
@@ -36,13 +36,13 @@ Run the following command to this end:
 ```
     python prepare_data.py --task=MRPC
     [--max_seq_length=128]
-    [--vocab_file=bert_pretrained_models/uncased_L-12_H-768_A-12/vocab.txt]
+    [--pretrained_model_name=bert-base-uncased]
     [--tfrecord_output_dir=data/MRPC] 
 ```
 
 - `--task`: Specifies the dataset name to preprocess. BERT provides default support for `{'CoLA', 'MNLI', 'MRPC', 'XNLI', 'SST'}` data.
 - `--max_seq_length`: The maxium length of sequence. This includes BERT special tokens that will be automatically added. Longer sequence will be trimmed. 
-- `--vocab_file`: Path to a vocabary file used for tokenization.
+- `--pretrained_model_name`: The name of a pre-trained model to load selected in the list of: `bert-base-uncased`, `bert-large-uncased`, `bert-base-cased`, `bert-large-cased`, `bert-base-multilingual-uncased`, `bert-base-multilingual-cased`, and `bert-base-chinese`.
```

Comment:
Polish:

The name of pre-trained BERT model. See the [doc](https://texar.readthedocs.io/en/latest/code/modules.html#texar.tf.modules.PretrainedBERTMixin) for all supported models.

Reply:
Please update the same part in the [Train and Evaluate](https://github.com/asyml/texar/tree/master/examples/bert#train-and-evaluate) section